src/modules/assetPack/reducer.ts

```diff
@@ -104,7 +104,8 @@ export const assetPackReducer = (state = INITIAL_STATE, action: AssetPackReducer
     case SAVE_ASSET_PACK_FAILURE: {
       return {
         ...state,
-        error: action.payload.error
+        error: action.payload.error,
+        loading: loadingReducer(state.loading, action)
```

Comment:
Not having this caused the loading state to never finish

Reply:
Good catch here, don't know how it wasn't there before 😅